assets/javascripts/kitten/components/cards/engagement-card/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import PropTypes from 'prop-types'
+import { Text } from '../../../components/typography/text'
+import { pxToRem } from '../../../helpers/utils/typography'
+import COLORS from '../../../constants/colors-config'
+
+const borderStyle = css`
+  box-shadow: 0 0 0 ${pxToRem(3)} ${COLORS.background1},
+    0 0 0 ${pxToRem(6)} ${({ hoverBorder }) => hoverBorder};
+`
+
+const StyledCard = styled.div`
+  border-radius: ${pxToRem(15)};
+  position: relative;
+  overflow: hidden;
+  background-repeat: no-repeat;
+  background-image: url(${({ imageSrc }) => imageSrc});
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  background-size: contain;
+  padding: ${pxToRem(10)};
+
+  ${({ as }) =>
+    (as === 'a' || as === 'button') &&
```

Comment:
On pourrait même l'écrire :
```suggestion
    ['button', 'a'].includes(as) &&
```
C'est un chouilla plus lisible :).